projects/1.18-fabric/assets/arcanus/arcanus/lang/zh_cn.json

```
@@ -0,0 +1,174 @@
+{
+	//-----Errors-----//
+	"error.arcanus.missing_spell": "这个法术不存在",
+	"error.arcanus.unknown_spell": "你还不知道这个法术",
+	"error.arcanus.not_enough_mana": "你没有足够的魔力来施放这个法术",
+	"error.arcanus.burnout": "你感觉有些疲惫……",
+
+	//-----Spells-----//
+	"spell.arcanus.lunge": "猛冲",
+	"spell.arcanus.dream_warp": "梦境扭曲",
+	"spell.arcanus.magic_missile": "魔法飞弹",
+	"spell.arcanus.telekinetic_shock": "隔空震荡",
```

```suggestion
	"spell.arcanus.telekinetic_shock": "念动攻击",
```
疑似游戏王梗
https://www.ygo-sem.cn/html/0/3/4/84847656.html